### DISCUSSION TITLE
Option to mark all received snaps as seen

### DISCUSSION THREAD

**Discussion by AlucardX14** [rodit/SnapMod#261] [Ideas & Feature Suggestions]
Useful for boosting snapscore

**Comment by RevealedSoulEven** [comment]
Yes it's really required as I recieve over thousands of snaps per day and get bored skipping them. Also an option to add blacklist would be better so that we could only see our friend's snap and all others will be marked as seen itself.

Btw hoping for the previous issues to be fixed before implementation of any new feature.
